user-modify/deprecated-webkit-user-modify-03.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>-webkit-user-modify should be unsupported in CSS.supports</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+@supports (-webkit-user-modify: read-only) {
+    #content {
+        width: 10px;
+        height: 10px;
+        background-color: red;
+    }
+}
+</style>
+<div id="content"></div>
+<script>
+    test(function() {
+        const content = document.getElementById('content');
+        assert_not_equals(content.style.backgroundColor, 'rgb(255, 0, 0)');
```

Comment:
This should use `getComputedStyle` instead.

Reply:
Thanks! Updated the test